keyboards/whitefox/keymaps/konstantin/config.h

```diff
@@ -18,3 +18,5 @@
 #define PERMISSIVE_HOLD
 #define TAPPING_TERM    200
 #define TAPPING_TOGGLE  2
+
+#define IS_COMMAND() (keyboard_report->mods == (MOD_BIT(KC_LSFT) | MOD_BIT(KC_RCTL)))
```

Comment:
```suggestion
```
It's already defined on lines 5 and 6 above, so remove these two lines. No idea how this passed CI.

Reply:
@drashna ↑